projects/go-lib/src/lib/components/go-search/go-search.component.ts

```diff
@@ -75,6 +80,18 @@ export class GoSearchComponent implements OnInit {
     }
   }
 
+  keyDown(event: Partial<KeyboardEvent>): void {
+    if (event.code === 'ArrowDown') {
+      this.goSearchService.arrowDownEvent.emit();
+    }
+  }
+
+  focusOnSearch(): void {
+    this.goSearchService.focusOnSearchEvent.subscribe(() => {
+      this.searchInput.nativeElement.focus();
+    });
+  }
```

Comment:
What was the purpose behind adding this here? Was it just to apply the correct css?

Reply:
See my comment below.